lib/src/main/java/uk/gov/di/ipv/core/library/helpers/RequestHelper.java

```diff
@@ -137,15 +137,16 @@ public static String getIpvSessionId(JourneyRequest request, boolean allowNull)
         return ipvSessionId;
     }
 
-    public static String getFeatureSet(Map<String, String> headers) {
-        String featureSet = RequestHelper.getHeaderByKey(headers, FEATURE_SET_HEADER);
-        if (featureSet == null) {
-            LOGGER.warn("{} not present in header", FEATURE_SET_HEADER);
-            return "default";
-        }
+    public static String getFeatureSet(JourneyRequest request) {
+        String featureSet = request.getFeatureSet();
+        LogHelper.attachFeatureSetToLogs(featureSet);
         return featureSet;
     }
 
+    public static String getFeatureSet(Map<String, String> headers) {
+        return RequestHelper.getHeaderByKey(headers, FEATURE_SET_HEADER);
```

Comment:
Should be attaching the feature set to the log context - or better, the ConfigService setter may be the best place for this. Let's discuss?